Add github-corners

trubesv opened the pull request:
Fixes #20 :)

ramnes commented:
Did you test it @trubesv? It doesn't appear on the doc. Does `html_sidebars` work with the RTD theme?

trubesv commented:
No I did not, I assumed it would work out of the box..

I added the templates path in #23 and switched to the Alabaster theme (which supports `html_sidebars`) in #24.

Sorry for this lack of rigor.
